test-apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,30 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+//I believe we are writing to servoPin, which I set to be PA_1
+CAN can1(PA_0, servoPin);
+float pulseWidthToSend;
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2):m_servoPwmOut(servoPin), m_servoRangeInDegrees(servoRangeInDegrees), m_minPulsewidthInMs(minPulsewidthInMs), m_maxPulsewidthInMs(maxPulsewidthInMs) {
```

Comment:
Default argument values are defined in the header, you can remove them from the .cpp file

Reply:
Fixed in 0cba407156933737ef1cbe0b85f325a7b83d9f0a